tests/bluetooth/ctrl_isoal/src/sub_sets/isoal_test_tx.c

```diff
@@ -2908,6 +2908,323 @@ ZTEST(test_tx_unframed, test_tx_unframed_1_sdu_1_frag_pdu_emit_err)
 					     ISOAL_STATUS_ERR_PDU_EMIT);
 }
 
+/**
+ * Test Suite  :   TX unframed SDU fragmentation
+ *
+ * Tests fragmentation of a single SDU contained in a single fragment
+ * into a single PDU such that it does not insert a skew into the stream.
+ */
+ZTEST(test_tx_unframed, test_tx_unframed_4_sdu_1_frag_4_pdu_stream_loc)
+{
+	struct tx_pdu_meta_buffer tx_pdu_meta_buf;
+	struct tx_sdu_frag_buffer tx_sdu_frag_buf;
+	uint32_t tx_sync_timestamp_expected;
+	struct isoal_pdu_buffer pdu_buffer;
+	isoal_source_handle_t source_hdl;
+	uint32_t tx_sync_offset_expected;
+	isoal_sdu_len_t sdu_total_size;
+	isoal_pdu_len_t pdu_write_size;
+	uint16_t tx_sync_seq_expected;
+	uint32_t stream_sync_delay;
+	uint64_t sdu_packet_number;
+	uint32_t tx_sync_timestamp;
+	uint32_t group_sync_delay;
+	uint8_t iso_interval_int;
+	uint64_t payload_number;
+	uint32_t tx_sync_offset;
+	uint32_t sdu_timestamp;
+	uint16_t testdata_indx;
+	uint16_t testdata_size;
+	uint16_t pdu_write_loc;
+	uint16_t sdu_read_loc;
+	uint64_t event_number;
+	uint32_t sdu_interval;
+	uint8_t sdu_fragments;
+	uint8_t testdata[53];
+	uint16_t tx_sync_seq;
+	uint32_t ref_point;
+	isoal_status_t err;
+	uint8_t max_octets;
+	uint8_t role;
+	uint8_t BN;
+	uint8_t FT;
+
+
+	/* Settings */
+	role = BT_CONN_ROLE_PERIPHERAL;
+	iso_interval_int = 1;
+	sdu_interval = CONN_INT_UNIT_US / 2;
+	max_octets = TEST_TX_PDU_PAYLOAD_MAX - 5;
+	BN = 2;
+	FT = 1;
+	stream_sync_delay = (iso_interval_int * CONN_INT_UNIT_US) - 200;
+	group_sync_delay = (iso_interval_int * CONN_INT_UNIT_US) - 50;
+
+	/* SDU Frag 1 --------------------------------------------------------*/
+	/* Sets initial fragmentation status */
+	isoal_test_init_tx_pdu_buffer(&tx_pdu_meta_buf);
+	isoal_test_init_tx_sdu_buffer(&tx_sdu_frag_buf);
+	init_test_data_buffer(testdata, 53);
+	pdu_buffer.handle = (void *)&tx_pdu_meta_buf.node_tx;
+	pdu_buffer.pdu = (struct pdu_iso *)tx_pdu_meta_buf.node_tx.pdu;
+	pdu_buffer.size = TEST_TX_PDU_PAYLOAD_MAX;
+	event_number = 2000;
+	sdu_packet_number = (event_number * BN);
+	sdu_timestamp = 9249;
+	ref_point = sdu_timestamp + (iso_interval_int * CONN_INT_UNIT_US) - 50;
+	sdu_total_size = 23;
+	testdata_indx = 0;
+	testdata_size = 23;
+	payload_number = event_number * BN;
+	pdu_write_loc = 0;
+	sdu_read_loc = 0;
+	pdu_write_size = 23;
+
+	source_hdl = basic_tx_test_setup(0xADAD,           /* Handle */
+					 role,             /* Role */
+					 false,            /* Framed */
```

Comment:
Because those aren't values that would have to change between test cases. The handle is not relevant the the tests so it only requires a place holder value and the tests for framed and unframed have different templates, so it's not possible to convert a framed test case to unframed or vice versa.